### PR TITLE
Ai photogram industrial 2

### PR DESCRIPTION
**Pull request by aizatibraimova**

### CODE REVIEW

**Review by aborov — APPROVED** [review]
awesome work!

File: app/models/like.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# Foreign Keys
+#
+#  fk_rails_...  (fan_id => users.id)
+#  fk_rails_...  (photo_id => photos.id)
+#
+class Like < ApplicationRecord
+  belongs_to :fan, class_name: "User", counter_cache: true
+  belongs_to :photo, counter_cache: true
+
+  validates :user_id, uniqueness: { scope: :photo_id, message: "has already liked this photo" }
```
Comment (thierrychau) [inline app/models/like.rb]: The foreign key is called `fan_id`, so I believe you need to validate on `fan_id` and not `user_id`

**Review by thierrychau — CHANGES_REQUESTED** [review]
Hi Aizat,

Great job on this first PR. I do not see your migrations for `devise` and `photos`, could you include them in this PR?

Everything else look good otherwise, besides a minor comment (see below)

**Review by thierrychau — APPROVED** [review]
Good to merge!

**Pull request merged**